src/Shared/Routes/ProductsPageRoutes.jsx

```diff
@@ -0,0 +1,14 @@
+import ProductPage from "../../Pages/DisplayProducts/Components/ProductPage/ProductPage";
+import DisplayProducts from "../../Pages/DisplayProducts/DisplayProducts";
+import { Routes, Route } from "react-router-dom";
```

Comment:
for importing we follow a pattern,

third-party package imports

then @ imports

then '../../'
example: 

import { Routes, Route } from "react-router-dom";
import ProductPage from "../../Pages/DisplayProducts/Components/ProductPage/ProductPage";
import DisplayProducts from "../../Pages/DisplayProducts/DisplayProducts";